pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -868,11 +867,15 @@ func (o *MustGatherOptions) createTempNamespace() (*corev1.Namespace, func(), er
 			o.PrinterDeleted.PrintObj(ns, o.LogOut)
 		}
 
-		if err := o.Client.RbacV1().ClusterRoleBindings().Delete(context.TODO(), crb.Name, metav1.DeleteOptions{}); err != nil {
-			fmt.Printf("%v\n", err)
-		} else {
-			o.PrinterDeleted.PrintObj(crb, o.LogOut)
-		}
+		// note that we can skip cluster role binding deletion here
```

Comment:
I think, we can remove the description in here as it does not pointing any code and people can see the history from PR?.

Reply:
I doubt that somebody will look at the clusterrolebinding change other than you and me :). The PR and the discussions in it seem to be self-explanatory and I'd prefer removing the comment in here.

Reply:
removed.